server/package.json

```diff
@@ -42,26 +42,40 @@
     "massive": "6.8.0",
     "minimist": "1.2.6",
     "node-cache": "5.1.2",
-    "node-pg-migrate": "5.9.0",
+    "node-pg-migrate": "6.2.2",
```

Comment:
Just a note to double check that the major version updates for this and uuid don't have something sneaky that we need to worry about. I'm sure its fine, especially since they're two packages that aren't used situationally, but figured I'd leave a note.